arch/xtensa/src/esp32/esp32_cpustart.c

```diff
@@ -193,6 +193,14 @@ void xtensa_appcpu_start(void)
 
   xtensa_attach_fromcpu0_interrupt();
 
+  /* Attach the timer interrupt */
+
+  irq_attach(XTENSA_IRQ_SWINT, (xcpt_t)xtensa_swint, NULL);
```

Comment:
Do we need to attach it a second time?
I believe since the IRQ number is the same attaching it only once is enough.